tests/integration/hive_metastore/test_pipeline_migrate.py

```diff
@@ -0,0 +1,91 @@
+from databricks.sdk.service.pipelines import PipelineLibrary, NotebookLibrary
+
+from databricks.labs.ucx.assessment.pipelines import PipelinesCrawler
+from databricks.labs.ucx.hive_metastore.pipelines_migrate import PipelinesMigrator, PipelineRule, PipelineMapping
+from ..assessment.test_assessment import _PIPELINE_CONF
+
+_TEST_STORAGE_ACCOUNT = "storage_acct_1"
+_TEST_TENANT_ID = "directory_12345"
+
+
+# pylint: disable=too-many-locals
+def test_pipeline_migrate(
+    ws,
+    make_pipeline,
+    make_notebook,
+    make_random,
+    watchdog_purge_suffix,
+    make_directory,
+    inventory_schema,
+    sql_backend,
+    runtime_ctx,
+):
+    src_schema = runtime_ctx.make_schema(catalog_name="hive_metastore")
+    target_schemas = 5 * [runtime_ctx.make_schema(catalog_name="hive_metastore")]
+
+    dlt_notebook_path = f"{make_directory()}/dlt_notebook.py"
+    src_table = runtime_ctx.make_table(catalog_name="hive_metastore", schema_name=src_schema.name, non_delta=True)
+    dlt_notebook_text = (
+        f"""create streaming table st1\nas select * from stream(hive_metastore.{src_schema.name}.{src_table.name})"""
+    )
+    make_notebook(content=dlt_notebook_text.encode("ASCII"), path=dlt_notebook_path)
+
+    pipeline_names = []
+    created_pipelines = []
+    for _ in range(3):
+        pipeline_names.append(f"pipeline-{make_random(4).lower()}-{watchdog_purge_suffix}")
+        created_pipelines.append(
+            make_pipeline(
+                configuration=_PIPELINE_CONF,
+                name=pipeline_names[_],
+                target=target_schemas[_].name,
+                libraries=[PipelineLibrary(notebook=NotebookLibrary(path=dlt_notebook_path))],
+            )
+        )
+
+    pipeline_crawler = PipelinesCrawler(ws=ws, sql_backend=sql_backend, schema=inventory_schema)
+    pipelines = pipeline_crawler.snapshot()
```

Comment:
```suggestion
    pipelines = runtime_ctx.pipelines_crawler.snapshot()
```
use context - it's already there.